src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
@niftylettuce @SandroMachado The native Android [Dialog](https://developer.android.com/reference/android/app/Dialog.html) has a `setCancelable` method. Since the new prop that I've added here only applies for Android, I thought it makes sense to name it after the native one. What do you think?